communitheme/meson.build

```diff
@@ -75,6 +75,7 @@ data_sources = [
   'ubuntu-logo-icon.svg',
   'ws-switch-arrow-down.svg',
   'ws-switch-arrow-up.svg',
+  'activities.svg'
```

Comment:
please ends it up with a comma, as the previous line, this enable to get a smaller diff when we append a new icon ;)